src/Model/Consignment/AbstractConsignment.php

```diff
@@ -756,6 +757,22 @@ public function setFullStreet(string $fullStreet): self
         return $this;
     }
 
+    /**
+     *
+     * @param string $barcode
+     * @param string $postalCode
+     * @param string $countryCode
+     *
+     * @return string
+     */
+    public function getBarcodeUrl(string $barcode, string $postalCode, string $countryCode): string
+    {
+        $barcodeUrl = (new TrackTraceUrl())
+            ->create($barcode, $postalCode, $countryCode);
```

Comment:
Dit is static
```
       $barcodeUrl = new TrackTraceUrl::create($barcode, $postalCode, $countryCode);
```
dit moet ook werken (vreemd dat dat andere wel gewoon werkte)

Reply:
Done